src/app/persistence/AttributePersistence.h

```diff
@@ -107,6 +148,35 @@ class AttributePersistence
     /// reason for the load failure).
     bool InternalRawLoadNativeEndianValue(const ConcreteAttributePath & path, void * data, const void * valueOnLoadFailure,
                                           size_t size);
+
+    /// Converts a DataModel::Nullable<T> to its storage representation.
+    template <typename T>
+    static void NullableToStorage(const DataModel::Nullable<T> & nullable,
```

Comment:
Attribute persistence static for these seem odd: they have no relation with the attribute persistence class.

Eventually these should be on on `DataModel::Nullable` or separate file.